client/js/admin.js

```diff
@@ -38,7 +39,86 @@ $(function() {
 			return false;
 		}
 	});
-})
+
+	$("#uploadTrigger").click(function(e) {
+		var form = $("#fileUpload")
+		form.off("change")
+		form.on("change", () => {
+			var XHR = new XMLHttpRequest()
+			XHR.open("POST", "/static/")
+			var fData = new FormData(form[0].form)
+			XHR.onload = function() {
+				var res = $(".upload-name.upload-item")
```

Comment:
Any reason to not simply `$(".upload-name.upload-item").removeClass("disabled").text(out.path)`?